backend/coins/btc/handlers/handlers.go

```diff
@@ -381,15 +391,26 @@ func (handlers *Handlers) getUTXOs(*http.Request) (interface{}, error) {
 }
 
 func (handlers *Handlers) getAccountBalance(*http.Request) (interface{}, error) {
+	type result struct {
+		Success   bool                   `json:"success"`
+		ErrorCode string                 `json:"errorCode,omitempty"`
+		Balance   map[string]interface{} `json:"balance,omitempty"`
```

Comment:
might as well make the Balance type concrete here instead of using an interface map, now that you are touching this and adding types :)

Reply:
Done :)